### DISCUSSION TITLE
ModuleNotFoundError at ._endpoint

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#767] [Q&A]
**Describe the bug**
stacktrace:
```


 File "falconpy\oauth2.py", line 47, in <module>
ModuleNotFoundError: No module named 'falconpy._endpoint'
code:
```

import time
from ._util import (
    perform_request,
    generate_b64cred,
    confirm_base_url,
    generate_error_result,
    autodiscover_region,
    )
from ._token_fail_reason import TokenFailReason
from ._endpoint._oauth2 import _oauth2_endpoints as Endpoints
```
```

tnx
nate

**Comment by ChristopherHammond13** [comment]
Hi @NSH531,  could you please expand on what you are trying to achieve? It is likely that there are some quirks in the way that your virtual environment is configured.

Furthermore, can I confirm that you are running a recent release of Python 3?
